src/Biconomy.js

```diff
@@ -807,12 +809,14 @@ async function handleSendTransaction(engine, payload, end) {
           metaTxApproach = smartContractMetaTransactionMap[contractAddr];
         }
 
-        let gasLimit = payload.params[0].gas;
+        let gasLimit = payload.params[0].gas || payload.params[0].gasLimit;
+        let txGas = payload.params[0].txGas;
```

Comment:
why gasLimit was not used for this purpose?

Reply:
ideally .gas should work. but I noticed when you send through ethers with txParam object, it doesnt recognise gas but gasLimit does. so I have added an OR

Reply:
txGas is part of request object in executeEIP712() in Biconomy Forwarder
so gasLimit would be something user intended to pass for whole transaction
and txGas is req.txGas thats optional custom value